src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -861,15 +865,12 @@ Scene* svgSceneBuild(SvgLoaderData& loaderData, Box vBox, float w, float h, Aspe
         docNode->translate(-vBox.x, -vBox.y);
     }
 
-    auto viewBoxClip = Shape::gen();
-    viewBoxClip->appendRect(0, 0, w, h);
-
-    auto compositeLayer = Scene::gen();
-    compositeLayer->composite(std::move(viewBoxClip), CompositeMethod::ClipPath);
-    compositeLayer->push(std::move(docNode));
-
     auto root = Scene::gen();
-    root->push(std::move(compositeLayer));
+    root->push(std::move(docNode));
+
+    float a, b, c, d;
+    root->bounds(&a, &b, &c, &d);
```

Comment:
Is this code we need?

Reply:
nope! these are leftovers after me checking the bounds, thx!